apache-apisix/assets/dashboards/apache-apisix_overview.json

```diff
@@ -0,0 +1,480 @@
+{
+    "title": "Apache APISIX Overview",
+    "description": "## Apache APISIX Overview\nApache APISIX integration with Datadog lets you dive deeper into your monitoring runs, helping you stay on top of your API's throughput, latency and more.",
+    "widgets": [
+        {
+            "id": 3577777298296914,
+            "definition": {
+                "type": "image",
+                "url": "/static/images/logos/apache-apisix_small.svg",
```

Comment:
qq: I might be missing context but I don't see this asset anywhere [?]
Better to give some cdn link, no?

Reply:
Sorry, it is displaying properly now, but I can't find the documentation for configuring logos. Maybe I need some help.

![image](https://github.com/DataDog/integrations-extras/assets/79972061/84829f75-f53e-4011-8c95-64ca61bf2628)

Reply:
> Sorry, it is displaying properly now

Okay, I assume the screenshot is from your local build output?

Reply:
Datadog is a cloud platform and cannot be built locally.

Reply:
If you need to verify, you can import the JSON file into the Datadog dashboard.